mmdet/ops/masked_conv/masked_conv.py

```diff
@@ -6,13 +6,15 @@
 from torch.autograd.function import once_differentiable
 from torch.nn.modules.utils import _pair
 
+from mmdet.utils import add_domain
+
 from . import masked_conv2d_ext
 
 
 class MaskedConv2dFunction(Function):
     @staticmethod
     def symbolic(g, features, mask, weight, bias, padding=0, stride=1):
-        return g.op('MaskedConv2d', features, mask, weight, bias, pad_i=padding, stride_i=stride)
+        return g.op(add_domain('MaskedConv2d'), features, mask, weight, bias, pad_i=padding, stride_i=stride)
```

Comment:
Does OpenVINO support Masked convolution? If not, I guess so dummy domain name should be used.

Reply:
No, OpenVINO does not support MaskedConv2d. This operation is implemented in the mmcv.